pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1508,4 +1508,9 @@ self: super: {
   # Test suite requires running a database server. Testing is done upstream.
   hasql-pool = dontCheck super.hasql-pool;
 
+  webify = appendPatch super.webify (pkgs.fetchpatch {
+    url = "https://github.com/ananthakumaran/webify/pull/27/commits/6d653e7bdc1ffda75ead46851b5db45e87cb2aa0.patch";
+    sha256 = "sha256:0xbfhzhzg94b4r5qy5dg1c40liswwpqarrc2chcwgfbfnrmwkfc2";
+  });
```

Comment:
Please add a comment here for why this patch is needed, and when you expect we can remove the patch.

Reply:
Done